client/extensions/tga-sign-off/src/components/fields/editor.tsx

```diff
@@ -48,13 +33,11 @@ export class UserSignOffField extends React.Component<IEditorProps, IState> {
         confirm('Are you sure you want to remove this publishing sign off?', 'Remove publishing sign off')
             .then((response) => {
                 if (response) {
-                    this.props.setValue({
-                        consent_publish: false,
-                        consent_disclosure: false,
-                        user_id: null,
-                        funding_source: null,
-                        affiliation: null,
-                        sign_date: null,
+                    superdesk.entities.article.patch(this.props.item, {
```

Comment:
Why doesn't `this.props.setValue` work here anymore? Just wondering why this change, and why client-core needed 360info (aka tga) specific code